pokemongo_bot/cell_workers/recycle_items_worker.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+from pokemongo_bot import logger
+
+
+class RecycleItemsWorker(object):
+
+    def __init__(self, bot):
+        self.api = bot.api
+        self.bot = bot
+        self.position = bot.position
+        self.config = bot.config
+        self.item_list = bot.item_list
+
+    def work(self):
+        self.bot.update_inventory()
+        if self.config.recycle_items:
```

Comment:
I would remove this.  When a worker is told to work, it should do it.  Leave the responsibility of if this worker recycles or not up to the caller, not the function